### PR TITLE
Add new mounts and outfits

### PR DESCRIPTION
**Pull request by Kamenuvol**

### CODE REVIEW

File: data/XML/mounts.xml [hunk 1]
```diff
@@ -16,31 +16,31 @@
...
-	<mount id="21" clientid="406" name="King Scorpion" speed="20" premium="yes" />
+	<mount id="21" clientid="406" name="Scorpion King" speed="20" premium="yes" />
 	<mount id="22" clientid="421" name="Rented Horse" speed="20" premium="no" />
 	<mount id="23" clientid="426" name="Armoured War Horse" speed="20" premium="yes" />
 	<mount id="24" clientid="427" name="Shadow Draptor" speed="20" premium="yes" />
 	<mount id="25" clientid="437" name="Rented Horse" speed="20" premium="no" />
 	<mount id="26" clientid="438" name="Rented Horse" speed="20" premium="no" />
-	<mount id="27" clientid="447" name="Ladybug" speed="20" premium="yes" />
-	<mount id="28" clientid="450" name="Manta" speed="20" premium="yes" />
+	<mount id="27" clientid="447" name="Lady Bug" speed="20" premium="yes" />
```
Comment (Kamenuvol) [inline data/XML/mounts.xml]: I used some videos to see the names. I've seen "Lady Bug" in:
https://youtu.be/-MG3OaMyr-4?t=11m8s
https://youtu.be/drDK51XIt38?t=4m56s

Comment (raymondtfr) [inline data/XML/mounts.xml]: That's weird. I thought so because the creature name is ladybug,

**Comment by marksamman** [comment]
Thanks!

File: data/XML/mounts.xml (hunk 1 above)
Comment (vankk) [inline data/XML/mounts.xml]: Look the example of Titanica, the creature name is Crustacea Gigantica and the mount is Titanica.

**Pull request merged**